cassandra/cqltypes.py

```diff
@@ -121,6 +122,76 @@ def __new__(metacls, name, bases, dct):
 ))
 
 
+def cqltype_to_python(cql_string):
+    """
+    Given a cql type string, creates a list that can be manipulated in python
+    Example:
+        int -> ['int']
+        frozen<tuple<text, int>> -> ['frozen', ['tuple', ['text', 'int']]]
+    """
+    scanner = re.Scanner((
+        (r'[a-zA-Z0-9_]+', lambda s, t: "'{}'".format(t)),
+        (r'<', lambda s, t: ', ['),
+        (r'>', lambda s, t: ']'),
+        (r'[, ]', lambda s, t: t),
+        (r'".*?"', lambda s, t: "'{}'".format(t)),
+    ))
```

Comment:
Interesting idea. I like the fact that everything inside  a `<>` are encapsulated as a python list.